### PR TITLE
Add nexus list to settings page

### PR DESCRIPTION
**Pull request by 0x5378**
Added nexus list and copy to settings page

Built a way for WooCommerce API keys to be regenerated

Clean TaxJar API key of whitespace

### CODE REVIEW

**Comment by fastdivision** [comment]
Looking good :+1:

File: includes/class-wc-taxjar-nexus.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        } else {
+          if ( isset ( $nexus->country ) ) {
+            $desc_text .= $nexus->country;
+          }
+        } 
+      }
+
+      $desc_text .= "<br><br><a href='" . $this->integration->regions_uri . "' target='_blank'>Add or update nexus locations</a>";
+    } else {
+      $desc_text .= "<p>TaxJar needs to your business locations in order to calculate sales tax properly. Please add them <a href='" . $this->integration->regions_uri . "' target='_blank'>here</a>.<p>";
```
Comment (fastdivision) [inline includes/class-wc-taxjar-nexus.php]: @0x5378 TaxJar needs ~~to~~ your business locations

**Comment by fastdivision** [comment]
@0x5378 Tested on WooCommerce 2.4, nexus list and API key regeneration is working for me :shipit:

**Comment by 0x5378** [comment]
:shipit:

**Pull request closed without merging**